pcsx2/gui/AppCoreThread.cpp

```diff
@@ -357,10 +360,11 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	wxString gameName;
 	wxString gameCompat;
 	wxString gameMemCardFilter;
+	wxString cheats_ws_archive = Path::Combine(PathDefs::GetProgramDataDir(), wxFileName(L"cheats_ws.zip"));
 
-	int numberLoadedCheats;
-	int numberLoadedWideScreenPatches;
-	int numberDbfCheatsLoaded;
+	int numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats");
```

Comment:
I made the code changes in a bit of a hurry., I probably made lots of other stupid mistakes, I'll look into them eventually with proper testing.